.github/workflows/stale.yml

```diff
@@ -13,7 +13,8 @@ jobs:
     - uses: actions/stale@v1
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-pr-message: "This PR has been inactive for long enough to be automatically marked as stale. This means it is at risk of being closed by a maintainer if it is not updated or reviews are not addressed. If your PR is closed as stale, feel free to open a new one after dealing with the issues. This may also be an indication that the maintainers do not have interest in this change, you can try to convince them otherwise, or persist in the doomed world you have created."
+        stale-pr-message: "This PR has been inactive for long enough to be automatically marked as stale. This means it is at risk of being auto closed in ~ 7 days, please address any outstanding review items and ensure your PR is finished, if these are all true and you are auto-staled anyway, you need to actively ask maintainers if your PR will be merged. Once you have done any of the previous actions then you should request a maintainer remove the stale label on your PR, to reset the stale timer"
         days-before-stale: 7
+        days-before-close: 7
```

Comment:
I'm happy for maintainers to discuss increasing this value @tgstation/commit-access 

But I think we should only allow a maximum of two weeks (14 days) grace period.

Reply:
It's fine as is IMO, it's not like closing makes the code irrecoverable.

I'd rather it close earlier so that way players aren't updating it for merge conflicts and such when no one was planning to look at it anyways 😏